cmd/root.go

```diff
@@ -258,7 +267,10 @@ func init() {
 	rootCmd.PersistentFlags().String("run-mode", "release", "Set the run mode for this service, 'release', 'debug' or 'test'. Defaults to 'release'.")
 
 	// Bind these to viper
-	viper.BindPFlags(rootCmd.PersistentFlags())
+	err = viper.BindPFlags(rootCmd.PersistentFlags())
+	if err != nil {
+		log.WithError(err).Fatal("could not bind flags to viper")
+	}
```

Comment:
```suggestion
	cobra.CheckErr(viper.BindPFlags(rootCmd.PersistentFlags()))
```

cobra provides a short-cut for fatal viper initialisation errors. These always indicate programming errors and should be caught during testing.